plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/preferences/RunOptionsDefaultsComponent.java

```diff
@@ -670,4 +675,9 @@ void join() throws InterruptedException {
     }
     projectInput.join();
   }
+
+  public void setAccountRequired(boolean required) {
```

Comment:
dosn't need to be public

Reply:
It needs to be public. this method called by  com.google.cloud.tools.eclipse.dataflow.ui.launcher.DefaultedPipelineOptionsComponent.setAccountRequired(boolean)